mm2src/coins_activation/src/init_token.rs

```diff
@@ -90,11 +91,13 @@ where
     InitTokenError: From<Token::ActivationError>,
     (Token::ActivationError, InitTokenError): NotEqual,
 {
+    // Todo: we should check if the contract has been enabled before in case the user is re-enabling the coin using a different ticker
```

Comment:
This sounds like an EVM specific logic, why checking it here?

Reply:
Just a misplaced todo, will move it to a better place.